tsconfig.json

```diff
@@ -9,7 +9,7 @@
 		"outDir": "_build/",
 		"removeComments": false,
 		"sourceMap": true,
-		"target": "es6",
+		"target": "es5",
```

Comment:
Was there a reason why we did this?  Just asking, as there likely is a good one.